testsuite/initializationfunctions.ml

```diff
@@ -37,6 +37,7 @@ let _ =
   printf "%s\n" (rounding_to_string x);
   printf "%d\n" (M.get_prec x);
   (* Should not raise Precision_range exception *)
-  let _ = M.make_zero ~prec:M.mpfr_prec_min M.Positive in ();
+  let _ = M.make_zero ~prec:M.mpfr_prec_min M.Positive in
```

Comment:
Please keep the original style when no changes are introduced, thanks!

Reply:
👍